tests/bazel_tests.bzl

```diff
@@ -105,11 +106,17 @@ def _bazel_test_script_impl(ctx):
   build_file = go.declare_file(go, path="BUILD.in")
   ctx.actions.write(build_file, ctx.attr.build)
 
+  extra_files = ""
+  for extra in ctx.attr.extra_files:
```

Comment:
Quote filenames using `shell.quote` in `go/private/skylib/lib/shell.bzl`. Also, join paths using `paths.join` in the same directory.

It's best to build a list of strings (or a list of files that are converted to strings in a list comprehension), then join them with `" ".join(...)`.

Reply:
Done.